heureka/ui/package.json

```diff
@@ -32,7 +32,7 @@
     "immer": "^10.0.0",
     "jest": "^29.3.1",
     "jest-environment-jsdom": "^29.3.1",
-    "juno-ui-components": "https://assets.juno.global.cloud.sap/libs/juno-ui-components@2.13.8/package.tgz",
+    "@cloudoperators/juno-ui-components": "^2.15.4",
```

Comment:
please move it to dependencies

Also move and update 
- communicator
- messages-provider
- url-state-provider